### ISSUE TITLE
[warn] Unable to notify Sentry

### ISSUE THREAD

**Issue by scottmessinger** [getsentry/sentry-elixir#184]
When I add this line to my channel action: `Sentry.Context.set_extra_context(%{payload: "hi"})` and then trigger an error with `1 = 2`, I get this error in the console:

```elixir
backend  │ [warn] Unable to notify Sentry due to %FunctionClauseError{arity: 3, function: :pop, module: Keyword}! [initial_call: {Phoenix.Channel.Server, :init, [:Argument__1]}, pid: #PID<0.1161.0>, registered_name: [], error_info: {:exit, {{:badmatch, 2}, [{Spikey.SessionChannel, :"-handle_in/3-fun-2-", 1, [file: 'web/channels/session_channel.ex', line: 71]}, {Appsignal.Phoenix.Channel, :channel_action, 4, [file: 'lib/appsignal/phoenix/channel.ex', line: 60]}, {Phoenix.Channel.Server, :"-handle_info/2-fun-4-", 4, [file: 'lib/phoenix/channel/server.ex', line: 226]}, {:gen_server, :try_dispatch, 4, [file: 'gen_server.erl', line: 601]}, {:gen_server, :handle_msg, 5, [file: 'gen_server.erl', line: 667]}, {:proc_lib, :init_p_do_apply, 3, [file: 'proc_lib.erl', line: 247]}]}, [{:gen_server, :terminate, 7, [file: 'gen_server.erl', line: 812]}, {:proc_lib, :init_p_do_apply, 3, [file: 'proc_lib.erl', line: 247]}]}, ancestors: [#PID<0.1159.0>], messages: [], links: [#PID<0.1159.0>, #PID<0.339.0>], dictionary: [sentry_context: %{extra: %{payload: "hi"}}], trap_exit: false, status: :running, heap_size: 4185, stack_size: 27, reductions: 110318]
```

**Comment by mitchellhenke** [comment]
@scottmessinger thanks for finding this bug 💖

I've opened #185 to fix it, and will likely push a new release after it gets merged.

**Comment by mitchellhenke** [comment]
Closed in #185 and released in `5.0.1`
